proto/iotics/api/common.proto

```diff
@@ -20,11 +20,8 @@ option php_namespace = "Iotics\\Api";
 
 
 // Limit is a request parameter to limit the number of results.
-// The use of "Limit" is:
-// - if limit is not supplied or supplied with no limit value, assume default=500
-// - if limit is supplied assume max number of entries is whatever specified. The limit value is capped to 1000.
 message Limit {
-  // Max number of results
+  // Max number of results (under server max limit constraint)
```

Comment:
I guess the issue here is that "server max limit" usually means "for the local host called" .. except for remote search - so maybe we should add a note to search regarding the limit.

How about e.g.:

> Maximum number of results (up to at most the space-configured maximum). Note that for remote requests the maximum depends on the remote, not local space.

(I'm not 100% happy about this proposed bit - but should we talk about servers or hosts or spaces & I think it's worth referencing the fact that not all limits are based on the local/configured one. Maybe a generic "Some requests have other limits which are documented therein")